server/package-lock.json

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
add package-lock.json to gitignore please.
just package.json is enough.

Reply:
okay